src/sider/sykmeldinger/container/DinSykmeldingSide.tsx

```diff
@@ -34,7 +26,7 @@ export const getSykmelding = (
   });
 };
 
-const DinSykmeldingSide = (): ReactElement => {
+export function DinSykmeldingSide() {
```

Comment:
`export default function`, eller hva synes vi er best?

Fra ChatGPT:

> Summary: When to Use Each
Use export default when you have a single, primary component in a file and want simpler import syntax. This approach aligns with the convention of using default exports for main components in React files.
Use named export if:
You have multiple components or utilities in a file.
You want to ensure consistent naming across imports.
You need better tree-shaking support in your project for optimized bundling.
In practice, many teams use export default for primary components and named exports for additional helper functions or sub-components in the same file. This balanced approach keeps imports manageable while maintaining flexibility for complex files.

Reply:
Prøvde å sjekke både [airbnb sin styleguide](https://github.com/airbnb/javascript) og [react.dev](https://react.dev/learn/importing-and-exporting-components), men fant ingen tydelige guidelines på det. Helt whatever for meg 🤷‍♂️